### ISSUE TITLE
http site requires username and password

### ISSUE THREAD

**Issue by mnjhost** [xibosignage/xibo#108]
Hi, I love this software but I have one issue. I need to show some webpages that require a basic username and password. The types of pages are currently Nagvis pages, http://www.nagvis.org/
Now explorer did offer the option of using http://user:user@domain.com but as most now know this does not work. Can anyone think of how this can be done as I think many people will require this function. Hope you can help guys/girls and have a great weekend

Mike

**Comment by dasgarner** [comment]
Are you saying that user:password@domain doesn't work because Nagvis doesn't support that authentication method (basic authentication) and instead has a login form - much like the Xibo CMS itself?

If that is the case, I don't see any way Xibo can show content from that system without that system providing an alternative authentication method or API.

**Comment by mnjhost** [comment]
Yes thats correct disgarner, the sites have read only access but the access for the sites cannot be turned off. Even though once logged on you only see a map showing server status, it still requires a login. Shame that we cannot get a area in the http section that will allow you to enter a username and password. But hey the software Xibo rocks for sure.

**Comment by dasgarner** [comment]
Collecting and storing the information is no problem :smile - passing that to the 3rd party webpage automatically is a fair bit more complicated! It would need some client side plugin or something (i'm guessing)

**Comment by txantxiku** [comment]
I've got the same problem mnjhost, did you solved the problem? Did you found any pluggin? Thanks in advanced.

**Comment by mnjhost** [comment]
Hi Tcantxiku, what I ended up doing was using a browser add on and that was not that bad

Kr

**Comment by mnjhost** [comment]
It was called auto login for chrome, current version is 4.3.1.0

**Comment by txantxiku** [comment]
We installed it, and chrome works propertly in client PC. But when we visualize with Xibo, still appears user and password screen. Did you do something else?
Is possible in Xibo, activate chrome by defect navigator?

**Comment by txantxiku** [comment]
Thanks for your collaboration

**Comment by mnjhost** [comment]
I will post you some info over the weekend. I got it working doing something on both the server and client side.

**Comment by txantxiku** [comment]
Please when you get a bit of time, send me the information. Thanks in advanced.

**Comment by markreitzel** [comment]
mnjhost,    I know this is an old thread.  Wondering how you were able to get a web page with authentication operating in Xibo.

**Comment by dasgarner** [comment]
We now use GitHub as a verified bug tracker only - if you'd like to ask a question please use the [community forum](https://community.xibo.org.uk)
